index.js

```diff
@@ -248,3 +250,37 @@ function generateImage (opt, callback) {
     callback(null, container);
   });
 }
+
+function RoundAllValue (obj, decimal = 0) {
+  for (var key in obj) {
+    if (obj.hasOwnProperty(key)) {
```

Comment:
Use 
```js
Object.keys(obj).forEach(key => {
    ...
}
```
 instead of 
```js
for (var key in obj) {
    if (obj.hasOwnProperty(key){
        ...
    }
}
```